fs/test/tests_test.go

```diff
@@ -46,4 +48,54 @@ func TestTestFuncs(t *testing.T) {
 		_, err = os.Open(path)
 		So(err, ShouldNotBeNil)
 	})
+
+	Convey("We can mock stdin", t, func() {
+		origStdin, stdinWriter, err := mockStdinRW("test")
+		So(origStdin, ShouldNotBeNil)
+		So(stdinWriter, ShouldNotBeNil)
+		So(err, ShouldBeNil)
+
```

Comment:
There's no test here that "test" ends up in STDIN. Read from os.Stdin to confirm.